test_scripts/API/SetGlobalProperties/017_ATF_TTS_IsReady_no_resp_SetGlobalProperties_TTS_SetGlobalProperties_gets_UNSUP_RES.lua

```diff
@@ -0,0 +1,163 @@
+        EXPECT_HMICALL("BasicCommunication.ActivateApp")
+        :Do(function(_,data1)
+          self.hmiConnection:SendResponse(data1.id,"BasicCommunication.ActivateApp", "SUCCESS", {})
+        end)
+      end)
+    end
+  end)
+  EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "FULL", systemContext = "MAIN"})
+end
+
+commonSteps:PutFile("Precondition_PutFile", "icon.png")
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+
+function Test:TestStep_SetGlobalProperties_WARNINGS_to_UI_SGP_and_UNSUPPORTED_RESOURCE_to_TTS_SGP()
+  local cid = self.mobileSession:SendRPC("SetGlobalProperties",
+  {
+    menuTitle = "Menu Title",
+    timeoutPrompt = {{ text = "Timeout prompt", type = "TEXT" }},
+    vrHelp =
+      {{
+        position = 1,
+        image = { value = "action.png", imageType = "DYNAMIC"},
+        text = "VR help item"
+    }},
+    menuIcon = { value = "action.png", imageType = "DYNAMIC" },
+    helpPrompt = {{ text = "Help prompt", type = "TEXT" }},
+    vrHelpTitle = "VR help title",
+    keyboardProperties = { keyboardLayout = "QWERTY", language = "EN-US"}
+  })
+
+  EXPECT_HMICALL("TTS.SetGlobalProperties",
+  {
+    timeoutPrompt = {{ text = "Timeout prompt", type = "TEXT" }},
+    helpPrompt = {{ text = "Help prompt", type = "TEXT" }},
+    appID = self.applications[config.application1.registerAppInterfaceParams.appName]
+  })
+  :Do(function(_,data) self.hmiConnection:SendResponse(data.id, "TTS.SetGlobalProperties", "UNSUPPORTED_RESOURCE", {info = "unsupported resource"}) end)
+
+  EXPECT_HMICALL("UI.SetGlobalProperties",
+  {
+    vrHelpTitle = "VR help title",
+    vrHelp = {{ position = 1, text = "VR help item" }},
+    menuTitle = "Menu Title",
+    menuIcon = { imageType = "DYNAMIC"},
+    keyboardProperties = { keyboardLayout = "QWERTY", language = "EN-US" },
+    appID = self.applications[config.application1.registerAppInterfaceParams.appName]
+  })
+  :ValidIf(function(_,data)
+    local value_Icon = storagePath .. "action.png"
+    if (string.match(data.params.menuIcon.value, "%S*" .. "("..string.sub(storagePath, 2).."action.png)" .. "$") == nil ) then
+      print("\27[31m value of menuIcon is WRONG. Expected: ~".. value_Icon .. "; Real: " .. data.params.menuIcon.value .. "\27[0m")
+      return false
+    else
+      return true
+    end
+  end)
+  :ValidIf(function(_,data)
+    local value_Icon = storagePath .. "action.png"
+    if (string.match(data.params.vrHelp[1].image.value, "%S*" .. "("..string.sub(storagePath, 2).."action.png)" .. "$") == nil ) then
+      print("\27[31m value of vrHelp.image is WRONG. Expected: ~".. value_Icon .. "; Real: " .. data.params.vrHelp[1].image.value .. "\27[0m")
+      return false
+    else
+      return true
+    end
+  end)
+  :Do(function(_,data) self.hmiConnection:SendResponse(data.id, "UI.SetGlobalProperties", "WARNINGS", {}) end)
+
+  EXPECT_RESPONSE(cid, { success = true, resultCode = "UNSUPPORTED_RESOURCE", {info = "unsupported resource"}})
```

Comment:
@istoimenova , script was updated in commit 522b81248a02956192f11ccb19f342995d5fe4a7